pkg/operator/flags.go

```diff
@@ -35,6 +35,7 @@ const (
 	GuardrailsDeployManaged            = "aro.guardrails.deploy.managed"
 	CloudProviderConfigEnabled         = "aro.cloudproviderconfig.enabled"
 	EtcHostsEnabled                    = "aro.etchosts.enabled"
+	EtcHostsManaged                    = "aro.etchosts.managed"
```

Comment:
This is just a suggestion, but how about leaving comments how it will work when we enable them.